admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/createtenantwizard/template.vue

```diff
@@ -249,7 +270,7 @@
                     }
                 }
 
-                return axios.get('/perapi/admin/sites/name/available?name=' + value)
+                return nameAvailableDebouncer.call(() => axios.get('/perapi/admin/sites/name/available?name=' + value), 375)
```

Comment:
Calling `axios.get` in this way looks a bit unusual for this project. It's skipping the API layer. Typically we see functions inside templates calling the api, and then apiImpl calls some variation on fetch to call the API. What we have here is simpler and more direct, but I don't know if it's OK to deviate from the established patterns.

Reply:
agree with @cmrockwell we should go through the api here using axios directly may not include the correct session and may in general cause an issue if/when we switch away from axios.